pkg/apis/duck/v1alpha1/subscribable_types.go

```diff
@@ -28,7 +28,9 @@ import (
 type Subscribable struct {
 	// TODO: What is actually required here for Channel spec.
 	// This is the list of subscriptions for this channel.
-	Subscribers []ChannelSubscriberSpec `json:"subscribers,omitempty"`
+	// +patchMergeKey=ref
```

Comment:
Could you maybe add a comment here about the bug so it doesn't get changed (especially since the tests don't seem to test anything).